### ISSUE TITLE
short duration 1k training data

### ISSUE THREAD

**Issue by zillur01** [lucas-ventura/chapter-llama#10]
Hi, thanks for this excellent work with detailed reproduction instructions.  i was wondering if i can get the 1k short-duration training set you used for ablation study (Tab A10, A6). The VLM model I am using only considers short-duration videos. I can probably download your 10k training data and only extract short videos from there and train your model and my model with that to compare.

also is this the validation subset you used mostly in the paper `sml300_val` instead of full 10k test set, in Tab 2, Tab 3, A6?

**Comment by lucas-ventura** [comment]
Hi, thanks for the kind words!

I’ve just updated the repository, you can do:

```bash
git pull
bash tools/download/docs.sh s1k-2_train
bash tools/download/docs.sh s1k_train
```

to get the short 1k subsets.
**`s1k-2_train`** was used to train the selector, and **`s1k_train`** to train Chapter-Llama.
Both are subsets of **`s10k-2_train`** and **`s10k_train`**, so all corresponding videos and captions should already be available on Hugging Face.

And yes, we report results on **`sml300_val`** in all tables except when comparing to VidChapters (Tables 1, A.13, and A.14).
